.github/workflows/pr.yaml

```
@@ -1,58 +1,33 @@
 defaults:
   run:
     shell: bash -x {0}
 
+env:
+  GCS_BUCKET: "gh-af"
+
 jobs:
   initial:
-    if: ${{ !contains( github.event.pull_request.labels.*.name, 'test:skip:pr') }}
-    uses: ./.github/workflows/initial-template.yaml
+    uses: ./.github/workflows/inc_initial.yaml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/inc_initial.yaml": open /home/jailuser/git/.github/workflows/inc_initial.yaml: no such file or directory

(workflow-call)